libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface HubspotFormField {
+    name: string;
+    value: string | any;
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    selectedValue?: string;
+    userEmail: string;
+    clientId: string | number;
+    dialogTitle: string;
+    cancelButtonText?: string;
+    showCancelButton?: boolean;
+    onClose: () => void;
+    onFormSubmitted: () => void;
+}
+
+/**
+ * @internal
+ */
+export const ConversionTouchPointDialogBase: React.FC<IConversionTouchPointDialogBaseProps> = (props) => {
+    const HUBSPOT_EMAIL_FIELD_NAME = "email";
+    const HUBSPOT_GACID_FIELD_NAME = "gacid";
+    const HUBSPOT_PORTAL_ID = "8934247";
```

Comment:
As `ConversionTouchPointDialogBase` is a component in the SDK, it can be used by different apps in different contexts, can't it?
※ AFAIK, [gooddata-ui-sdk](https://github.com/gooddata/gooddata-ui-sdk) is a public repo, meaning it can also be used by other customers.

If we hard-code `HUBSPOT_PORTAL_ID` as `8934247`, can it cause any issue if the component is used by other customers who probably have a different Hubspot portal ID or Hubspot form ID?
→Should we pass this value as a prop instead of hard-coding it like this?

Same concerns for `HUBSPOT_FORM_ID` below.

Reply:
I have added the props for these values. Thanks!